tools/restatectl/src/commands/metadata/put.rs

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2024 -  Restate Software, Inc., Restate GmbH.
+// All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use crate::commands::metadata::patch::{patch_value, PatchValueOpts};
+use crate::commands::metadata::MetadataCommonOpts;
+use anyhow::anyhow;
+use clap::Parser;
+use clap_stdin::FileOrStdin;
+use cling::{Collect, Run};
+use serde_json::Value;
+use std::collections::HashMap;
```

Comment:
nit: Our convention is to split imports into
<std>
blank
<third-party>
blank
<restate_*>
blank
<crate/super/self::*>